### PR TITLE
Update AWS Terminology

### PR DESCRIPTION
**Pull request by salvis2**
Part of https://github.com/yuvipanda/hubploy/pull/88 to have AWS-specific terminology. Just a quick edit.

### CODE REVIEW

File: deployments/myhub/hubploy.yaml [hunk 1]
```diff
@@ -11,8 +11,8 @@ images:
       service_key: gcr-key.json
     aws:
       # Pushes to Amazon ECR
-      project: # TODO: AWS account id
-      zone: # TODO: Zone in which your container image should live. Match your cluster's zone
+      account_id: # TODO: AWS account id
+      region: # TODO: Zone in which your container image should live. Match your cluster's zone
```
Comment (consideRatio) [inline deployments/myhub/hubploy.yaml]: The comment still reference "Zone", same for the changed lines below. I'm not familiar with AWS so I'll let you update this without a guess-suggestion from me.

Comment (salvis2) [reply deployments/myhub/hubploy.yaml]: Ah, they are supposed to be `region`. Forgot to check the comments. Thanks!

**Comment by salvis2** [comment]
As noted here: https://github.com/yuvipanda/hubploy/pull/88#discussion_r473370601, this will be a non-backward-compatible change for `hubploy-template`. Unsure if this comment is enough to help people figure out the issue if they find it. I don't think a lot of people use this repo to keep their repos up-to-date, so it shouldn't affect anyone on the template end. We will update the `hubploy` docs to note this. I think most people have pinned a specific `hubploy` version, so we would maybe just need a note in the Readme / docs to say that this is a minor breaking change?

**Comment by salvis2** [comment]
Adding in backwards-compatibility here: https://github.com/yuvipanda/hubploy/pull/88/commits/0a500e365604c3ab26d4fe6f51f01ea04400c6f9

**Pull request merged**